prompts/graphics-engineer-2.md

```diff
@@ -0,0 +1,44 @@
+# Graphics Engineer Take Home (verrsion 2)
+
+## Focus
+
+- threejs, webgl, shaders
+- ability to pickup mrjs
+
+## Prompt
+
+Using mrjs, create an interactive experience. Make sure it works well as a 2D website and also in headset (_this part might be harder since they might not have a headset yet_).
```

Comment:
quick question here on the in headset feasability

Reply:
If they don't have one that's a problem. I think we're gonna need to start targeting more people with existing XR experience.